milvus/metadata.csv

```diff
@@ -0,0 +1,387 @@
+milvus.datacoord.stored.rows_num,gauge,,,,Number of stored rows of healthy segment,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.bucket,count,,,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.count,count,,,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.sum,count,,millisecond,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.watched_dml_chanel_num,gauge,,,,The num of dml channel watched by datanode,0,milvus,,,
+milvus.datanode.autoflush_buffer_op.count,count,,,,Count of auto flush buffer operations,0,milvus,,,
+milvus.datanode.consume.bytes.count,count,,,,,0,milvus,,,
+milvus.datanode.consume.msg.count,count,,,,Count of consumed msg,0,milvus,,,
+milvus.datanode.consume.tt_lag_ms,gauge,,millisecond,,Now time minus tt per physical channel,0,milvus,,,
+milvus.datanode.encode_buffer_latency.bucket,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.count,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.sum,count,,millisecond,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.flowgraph_num,gauge,,,,Number of flowgraphs,0,milvus,,,
+milvus.datanode.flush.buffer_op.count,count,,,,Count of flush buffer operations,0,milvus,,,
+milvus.datanode.flush.req.count,count,,,,Count of flush request,0,milvus,,,
+milvus.datanode.flushed_data.rows.count,count,,,,Num of rows flushed to storage,0,milvus,,,
+milvus.datanode.flushed_data.size.count,count,,byte,,Byte size of data flushed to storage,0,milvus,,,
+milvus.datanode.msg.dispatcher_tt_lag_ms,gauge,,millisecond,,time.Now() sub dispatcher's current consume time,0,milvus,,,
+milvus.datanode.msg.rows.count,count,,,,Count of rows consumed from msgStream,0,milvus,,,
+milvus.datanode.save_latency.bucket,count,,,,Latency of saving flush data to storage,0,milvus,,,
+milvus.datanode.save_latency.count,count,,,,Latency of saving flush data to storage,0,milvus,,,
+milvus.datanode.save_latency.sum,count,,millisecond,,Latency of saving flush data to storage,0,milvus,,,
+milvus.diskann.range_search_iters.bucket,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.range_search_iters.count,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.range_search_iters.sum,count,,,,DISKANN range search iterations,0,milvus,,,
+milvus.diskann.search_hops.bucket,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann.search_hops.count,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann.search_hops.sum,count,,,,DISKANN search hops,0,milvus,,,
+milvus.diskann_bitset_ratio.bucket,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
+milvus.diskann_bitset_ratio.count,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
+milvus.diskann_bitset_ratio.sum,count,,,,DISKANN bitset ratio for search and range search,0,milvus,,,
```

Comment:
The description field should explain what the difference is between these three metrics.